test/v2/Makefile

```diff
@@ -1,11 +1,17 @@
-clean:
-	rm -rf bin 2>/dev/null || true
+# Find all Go files under load/
+# The build command will rebuild the binary if any Go files change.
+GO_SOURCES_LOAD := $(shell find load -name "*.go" -type f)
 
-build: clean
+build: $(GO_SOURCES_LOAD)
 	go build -o bin/load load/main/load_main.go
 
-test:
-	cd correctness && go test
-
+# Makefile doesn't allow forwarding of arguments, so we use ARGS. Call this as:
+# make generate-load ARGS="config/environment/preprod.json config/load/100kb_s-1mb-3x.json"
 generate-load: build
 	./bin/load $(ARGS)
+
+clean:
+	rm -rf bin 2>/dev/null || true
+
+test:
+	cd correctness && go test
```

Comment:
Just a heads up, I'm rebranding this test suite from `correctness tests` to `live tests`. Once my branch merges, this will need to change to `cd live`. I'll keep an eye on this. If your branch merges before mine, I'll fix it before my branch merges.

Reply:
Fixed in [11ec5a4](https://github.com/Layr-Labs/eigenda/pull/1722/commits/11ec5a4c7171b1d7bbebc7d446ba28c4e09cbf2d)